### PR TITLE
Update ci to use latest singularity image for testing

### PR DESCRIPTION
**Pull request by bendavid**

### CODE REVIEW

File: scripts/ci/run_with_singularity.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+SINGULARITY_BIND="/scratch,/eos,/cvmfs" singularity run /cvmfs/unpacked.cern.ch/gitlab-registry.cern.ch/bendavid/cmswmassdocker/wmassdevrolling\:v5 $@
```
Comment (davidwalter2) [inline scripts/ci/run_with_singularity.sh:2]: should we not use "latest" here instead of v5?

Comment (bendavid) [reply scripts/ci/run_with_singularity.sh:2]: So "latest" still points to v4.  v5 is rather pointed to by "testing"

This PR is intentionally being used to test v5 BEFORE we point "latest" to it.

Comment (davidwalter2) [reply scripts/ci/run_with_singularity.sh:2]: Ah okay, so this needs to be updated then whenever a new singularity becomes the new default, or like in this case, before.

**Comment by bendavid** [comment]
No,
There's still a remaining problem with the template errors/warnings to be fixed first.

**Pull request merged**